tests/unit/views/auditLaunch/auditBasicAuthPasswordForm.spec.js

```diff
@@ -0,0 +1,74 @@
+import {shallowMount} from '@vue/test-utils'
```

Comment:
Il faut faire un fichier de test par composant. Tu peux ensuite imbriquer des "it" dedans pour chaque champs si tu veux plutot que de faire un fichier de test par champ

Reply:
okay, j'ai regroupé les tests dans auditBasicAuthForm.spec.js